pulp_puppet_plugins/pulp_puppet/plugins/importers/metadata.py

```diff
@@ -169,20 +156,18 @@ def _extract_non_standard_json(filename, temp_dir):
         tgz = tarfile.open(name=filename)
         tgz.extractall(path=extraction_dir)
         tgz.close()
-    except Exception, e:
+    except Exception:
         raise InvalidTarball(filename), None, sys.exc_info()[2]
 
     try:
-        # Recursively look for the metadata file
-        metadata_file_dir = _find_file_in_dir(extraction_dir, constants.MODULE_METADATA_FILENAME)
-
-        if metadata_file_dir is None:
+        # Attempt to find the metadata in the first folder of the tar.gz
+        module_dir = os.listdir(extraction_dir)[0]
+        metadata_filename = constants.MODULE_METADATA_FILENAME
+        metadata_full_path = os.sep.join([extraction_dir, module_dir, metadata_filename])
```

Comment:
How about ``os.path.join`` instead? I've never used ``os.sep`` directly, but I see that its documentation recommends using ``os.path.join``.